language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/builder/FunctionCompletionItemBuilder.java

```diff
@@ -129,6 +129,28 @@ public static CompletionItem build(ClassSymbol typeDesc, InitializerBuildMode mo
         return item;
     }
 
+    /**
+     * Creates and returns a completion item.
+     *
+     * @param functionSymbol BSymbol
+     * @param context        LS context
+     * @param symbolName     Name of the symbol
+     * @return {@link CompletionItem}
+     */
+    public static CompletionItem buildMethod(FunctionSymbol functionSymbol, BallerinaCompletionContext context,
+                                             String symbolName) {
+        CompletionItem item = new CompletionItem();
+        setMeta(item, functionSymbol, context);
+        if (functionSymbol != null) {
+            String funcName = functionSymbol.getName().get();
```

Comment:
We can use `functionSymbol.getName().orElse("name")` or having a isPresent check is safe